cmds/nspawn

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+nspawn_list() {
+    local path=${1}
+
+    ls -1 "${path}"
+}
+
+nspawn_create() {
+    local path=${1}
+    local name=${2}
+    local file=${3}
+
+    if [ -z "${name}" ] || [ -z "${file}" ]; then
+        echo "Error: empty parameter" >&2
+        echo "create_container name Dockerfile" >&2
```

Comment:
I understand these are placeholders, but the Docker references should be removed to avoid confusion.